android/src/main/java/io/invertase/firebase/notifications/RNFirebaseBackgroundNotificationReceiver.java

```diff
@@ -0,0 +1,47 @@
+package io.invertase.firebase.notifications;
+
+import android.app.ActivityManager;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.util.Log;
+
+import com.facebook.react.HeadlessJsTaskService;
+
+import java.util.List;
+
+public class RNFirebaseBackgroundNotificationReceiver extends BroadcastReceiver {
```

Comment:
Can we rename this to `RNFirebaseBackgroundNotificationActionReceiver` to make it clear that it handles actions and not notifications themselves

Reply:
Done

Reply:
👍